Pear/Profile/MatchProfileViewController.swift

```diff
@@ -0,0 +1,230 @@
+        profileTableView.register(ProfileSectionTableViewCell.self, forCellReuseIdentifier: ProfileSectionTableViewCell.reuseIdentifier)
+        profileTableView.register(ProfilePromptTableViewCell.self, forCellReuseIdentifier: ProfilePromptTableViewCell.reuseIdentifier)
+        profileTableView.rowHeight = UITableView.automaticDimension
+        profileTableView.bounces = true
+        profileTableView.separatorStyle = .none
+        profileTableView.estimatedSectionHeaderHeight = 0
+        profileTableView.sectionHeaderHeight = UITableView.automaticDimension
+        profileTableView.contentInset = UIEdgeInsets(top: 40, left: 0, bottom: 150, right: 0)
+        profileTableView.showsVerticalScrollIndicator = false
+        view.addSubview(profileTableView)
+
+        reachOutButton.backgroundColor = .backgroundOrange
+        reachOutButton.layer.cornerRadius = reachOutButtonSize.height / 2
+        reachOutButton.titleLabel?.font = ._20CircularStdBold
+        reachOutButton.setTitleColor(.white, for: .normal)
+        reachOutButton.addTarget(self, action: #selector(reachOutPressed), for: .touchUpInside)
+
+        switch chatStatus {
+        case .planning, .noResponses:
+            reachOutButton.setTitle("Reach out", for: .normal)
+        case .responding:
+            reachOutButton.setTitle("Pick a time", for: .normal)
+        default:
+            reachOutButton.setTitle("Send email", for: .normal)
+        }
+
+        view.addSubview(reachOutButton)
+    }
+
+    private func setupConstraints() {
+        let reachOutPadding = LayoutHelper.shared.getCustomVerticalPadding(size: 70)
+        let meetupPadding = 24
+        let meetupWidth: CGFloat = 319
+
+        meetupStatusView?.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(meetupPadding)
+            make.leading.equalTo(view.safeAreaLayoutGuide).inset(meetupPadding)
+            make.width.equalTo(meetupWidth)
+            make.height.equalTo(meetupStatusView?.getRecommendedHeight(for: meetupWidth) ?? 0)
+        }
+
+        if let meetupStatusView = meetupStatusView {
+            profileTableView.snp.makeConstraints { make in
+                make.leading.trailing.bottom.equalToSuperview()
+                make.top.equalTo(meetupStatusView.snp.bottom).offset(8)
+            }
+        } else {
+            profileTableView.snp.makeConstraints { make in
+                make.edges.equalToSuperview()
+            }
+        }
+
+        reachOutButton.snp.makeConstraints { make in
+            make.bottom.equalTo(view.safeAreaLayoutGuide).inset(reachOutPadding)
+            make.centerX.equalToSuperview()
+            make.size.equalTo(reachOutButtonSize)
+        }
+    }
+
+    @objc private func reachOutPressed() {
+        guard let pair = pair, let match = match, let user = user else { return }
+        let schedulingVC: SchedulingTimeViewController
```

Comment:
There's no reason to have this out here. Within each case just add `let` before `schedulingVC =`